tests/jobs.test.ts

```diff
@@ -151,6 +151,30 @@ describe('job queues', () => {
                 await waitForLogEntries(2)
                 expect(testConsole.read()).toEqual([['processEvent'], ['reply', 'runIn']])
             })
+
+            test.only('polls for jobs in future', async () => {
+                const DELAY = 10000 // 10s
```

Comment:
probably OK to wait with a smaller timeout to make tests go a bit faster

Reply:
Ha, definitely. Needed a big one only for S3.